### PR TITLE
Django upgrade

### PR DESCRIPTION
**Pull request by pwalczysko**
cc @sbesson 

the changes are on top of https://github.com/ome/prod-playbooks/pull/334

Deployed successfully on ome-training-3

Possibly there is a change in ``mapr`` in between ?
This here is a status from couple of days ago, with RC on mapr.

### CODE REVIEW

File: omero/training-server/playbook.yml [hunk 1]
```diff
@@ -387,12 +387,12 @@
     omero_server_datadir_managedrepo_mode: u=rwX,g=srwX,o=rX,+t
     omero_server_datadir_chown: False
     omero_server_release: "{{ omero_server_release_override | default('5.6.3') }}"
-    omero_web_release: "{{ omero_web_release_override | default('5.12.0') }}"
-    omero_figure_release: "{{ omero_figure_release_override | default('4.4.1') }}"
+    omero_web_release: "{{ omero_web_release_override | default('5.14.0.rc1') }}"
```
Comment (sbesson) [inline omero/training-server/playbook.yml:390]:
```suggestion
    omero_web_release: "{{ omero_web_release_override | default('5.14.0') }}"
```

**Review by sbesson — CHANGES_REQUESTED** [review]

**Comment by pwalczysko** [comment]
@sbesson thank you, changes applied as requested.
This is now applied on ``ome-training-3``
All works there, including mapr, parade, webtagging, tagsearch.
Checked the compliancy with the versions in this notebook

```
(venv3) [pwalczysko@ome-training-3 ~]$ pip freeze | grep web
omero-web==5.14.0
omero-webtagging-autotag==3.2.0
omero-webtagging-tagsearch==3.2.0
(venv3) [pwalczysko@ome-training-3 ~]$ pip freeze | grep mapr
omero-mapr==0.5.0
```

**Review by sbesson — APPROVED** [review]
Looks good and very straightforward. Happy for these changes to get deployed onto all training servers.

@jburel  should we capture the upgrade of the other UoD OMERO deployments as issues?

**Comment by pwalczysko** [comment]
This is now deployed on all 4 training servers. All works as expected.
Note that the manual delete of the ``...omero`` file was performed in each server.

```
cd /opt/omero/web/config/
sudo rm -rf omero-web-outreach-webapps.omero
```

**Pull request merged**